### PR TITLE
revert #318 - Remove Save Results Option

### PR DESCRIPTION
**Pull request by curquhart1**
revert #318 - Removing Pac Solution Check, Save Results Option, until the auth SPN policy is in place to handle calls to advisor/api/cdsanalysisrequest

### CODE REVIEW

File: src/tasks/checker/checker-v2/index.ts [hunk 1]
```diff
@@ -46,7 +46,6 @@ export async function main(): Promise<void> {
     useDefaultPAEndpoint: parameterMap['UseDefaultPACheckerEndpoint'],
     customPAEndpoint: parameterMap['CustomPACheckerEndpoint'],
     geoInstance: { name: "GeoInstance", required: false, defaultValue: undefined },
-    saveResults: parameterMap['SaveResults'],
```
Comment (swapnilbhalgat) [inline src/tasks/checker/checker-v2/index.ts:49]: saveResults is a required parameter, make it optional or pass default value i.e. false to it.

**Review by petrochuk — CHANGES_REQUESTED** [review]
Can you elaborate on this change?

File: src/tasks/checker/checker-v2/index.ts (hunk 1 above)
Comment (curquhart1) [reply src/tasks/checker/checker-v2/index.ts:49]: Ack

**Comment by petrochuk** [comment]
Did you test it with existing pipelines which have this task? I feel like it might be breaking change

**Comment by curquhart1** [comment]
> Did you test it with existing pipelines which have this task? I feel like it might be breaking change

This was a change that wasn't supposed to go to public because the backend wasn't set up properly. So any task with that option on wasn't working.

**Comment by petrochuk** [comment]
> > Did you test it with existing pipelines which have this task? I feel like it might be breaking change
> 
> This was a change that wasn't supposed to go to public because the backend wasn't set up properly. So any task with that option on wasn't working.

My question is still - did you test this potential breaking change? I didn't see any deployments of experimental version of this extension.

@JesseParsons we are getting close to releasing of new version. I don't think we are ready to take on this change now

**Comment by curquhart1** [comment]
> > > Did you test it with existing pipelines which have this task? I feel like it might be breaking change
> > 
> > 
> > This was a change that wasn't supposed to go to public because the backend wasn't set up properly. So any task with that option on wasn't working.
> 
> My question is still - did you test this potential breaking change? I didn't see any deployments of experimental version of this extension.
> 

We have only done the gulp ci unit tests, I was under the impression that it needed to be merged and pushed to DEV before we could test it with existing pipelines.

**Comment by petrochuk** [comment]
No, you can create "Experimental" version of this extension, publish it to market place and validate against existing pipelines

**Pull request closed without merging**